geode-docs/tools_modules/geode_for_redis.html.md.erb

```diff
@@ -35,13 +35,14 @@ Use gfsh to start at least one server with a command of the form:
 start server \
   --name=<serverName> \
   --locators=<locatorPort> \
-  --geode-for-redis-port=<geodeForRedisPort> \
-  --geode-for-redis-bind-address=<geodeForRedisBindAddress> \
-  --geode-for-redis-password=<geodeForRedisPassword>
+  --J=-Dgemfire.geode-for-redis-enabled=true \
+  --J=-Dgemfire.geode-for-redis-port=<geodeForRedisPort> \
+  --J=-Dgemfire.geode-for-redis-bind-address=<geodeForRedisBindAddress> \
+  --J=-Dgemfire.geode-for-redis-password=<geodeForRedisPassword>
 ```
 
-If any of the options `geode-for-redis-bind-address`, `geode-for-redis-password`, or `geode-for-redis-port`
-are included, a <%=vars.product_name%> server with <%=vars.product_name%> for Redis will be started.
+If the option `geode-for-redis-enabled`, is set to `true`, a <%=vars.product_name%> server with
```

Comment:
I think it is wrong to call "geode-for-redis-enabled" an "option". It used to be a gfsh option but is no longer. Consider replacing "option" with "gemfire property".